src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/NetworkFiles/smb_without_wins.conf

```diff
@@ -0,0 +1,252 @@
+#
+# Sample configuration file for the Samba suite for Debian GNU/Linux.
```

Comment:
Where did this file come from? What was it originally licensed as?

Reply:
I copied the content from the already existing file NetworkFiles/smb.conf

Reply:
Hmm, the existing smb.conf came with https://github.com/dotnet/corefx/pull/4211 . There is no mention about the origin of this file in that PR. We may have a pre-existing licensing issue.

> should I use the .NET foundation license header?

I do not know. We need to find the origin of this file first.

Reply:
samba repo is licensed as GPL 3.0. As far as I can tell, the sample configuration file is GPL 3.0 as well - I do not see anything to the contrary.

The non-product test code can be under GPL, but it needs to be attributed correctly https://github.com/dotnet/runtime/blob/main/docs/project/copyright.md#source-license